app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java

```diff
@@ -124,4 +268,13 @@ public void setCurrentPositionOffType(int currentPositionOffType) {
         int OFF_HAVE_SALARY_INSURANCE_COVERAGE = 1;
         int OFF_NO_SALARY = 2;
     }
+
+    //Day Session
+    @IntDef({
+            DaySession.AM, DaySession.PM
+    })
+    @interface DaySession {
+        int AM = 0;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.